o/opentelemetry-ruby/opentelemetry-ruby_ubi_8.5.sh

```diff
@@ -19,9 +19,9 @@
 # ----------------------------------------------------------------------------
 
 PACKAGE_NAME="opentelemetry-ruby"
-PACKAGE_VERSION=${1:-"opentelemetry-sdk-experimental/v0.1.0"}
+PACKAGE_VERSION=${1:-"opentelemetry-sdk-experimental/v0.3.2"}
 PACKAGE_URL="https://github.com/open-telemetry/opentelemetry-ruby.git"
-export RUBY_VERSION=${RUBY_VERSION:-2.7.0}
```

Comment:
@vinodk99 Check if we can make this available like GO version so we dont need to make hard coded value